resteasy-core/src/main/java/org/jboss/resteasy/plugins/providers/MultiValuedArrayParamConverter.java

```diff
@@ -0,0 +1,189 @@
+package org.jboss.resteasy.plugins.providers;
+
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Array;
+import java.lang.reflect.Method;
+
+import javax.annotation.Priority;
+import javax.ws.rs.ext.ParamConverter;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.resteasy.core.StringParameterInjector;
+import org.jboss.resteasy.resteasy_jaxrs.i18n.Messages;
+
+/**
+ * @author Marek Kopecky mkopecky@redhat.com
+ * @author Ron Sigal rsigal@redhat.com
+ */
+@Provider
+@Priority(Integer.MAX_VALUE)
+public class MultiValuedArrayParamConverter implements ParamConverter<Object>
+{
+   private Class<?> rawType;
+   private StringParameterInjector stringParameterInjector;
+   private String separator;
+
+   public MultiValuedArrayParamConverter(Class<?> rawType, Annotation[] annotations, StringParameterInjector stringParameterInjector, String separator)
```

Comment:
Quick clean up for `annotations `parameter that is not used here

Reply:
Ok, got it.